examples/official-storybook/stories/demo/welcome.stories.js

```diff
@@ -7,7 +7,9 @@ export default {
   component: Welcome,
 };
 
-export const ToStorybook = () => <Welcome showApp={linkTo('Button')} />;
+// export const ToStorybook = () => <Welcome showApp={linkTo('other-demo-buttonmdx--with-text')} />;
+// export const ToStorybook = () => <Welcome showApp={linkTo('Other/Demo/ButtonMdx')} />;
+export const ToStorybook = () => <Welcome showApp={linkTo('Other/Demo/Button')} />;
```

Comment:
Did you mean to check this in?

Reply:
Yeah, just to document other cases. Can delete